src/components/molecules/NavBar/NavBar.scss

```
@@ -9,6 +9,7 @@ $menu-item-size: $spacing--lg * 2;
 $logo-size: $spacing--xxl + $spacing--md;
 $volume-popover-width-with-sliders: 30em;
 $volume-popover-width-without-sliders: 15em;
+$spaceBackButton: 50px;
```

Before adding new variables/constants/any named elements,  surrounding ones can help you keep with the ad hoc convention:
```suggestion
$back-button-space: 50px;
```

In the above example:

- No camelCase
- No noun at first position 
  - logo size <- adjective noun
  - space back button <- is it the button or is it the space that is representing the value?